inbm/fpm/inb-provision-cloud/main.go

```diff
@@ -291,11 +292,15 @@ func createUnencryptedTemplate(templateDir string) string {
 }
 
 func getServerIp() string {
-	serverIp := promptString("\nPlease enter the server IP:")
-	if net.ParseIP(serverIp) == nil {
-		log.Fatalf("Invalid IP address provided.")
+	serverIPName := promptString("\nPlease enter the server IP or URL like http://xxx")
```

Comment:
It should actually be just a hostname.  It will error if you use http://.  For example, I enter 'proxy-dmz.intel.com' for the hostname when I test.

Reply:
@nmgaston addressed this. Just like how we have for the proxy